src/app/modules/registration/components/company-details/company-details.component.html

```diff
@@ -3,13 +3,14 @@
     <form [formGroup]="formGroup">
       <dynamic-ng-bootstrap-form [group]="formGroup"
                                  [model]="formModel"
+                                 [layout]="formLayout"
                                  (change)="onChange($event)">
       </dynamic-ng-bootstrap-form>
     </form>
-    <vc-registration-company-address></vc-registration-company-address>
+    <vc-registration-company-address (formWasChanged)="onChildFormChange($event)"></vc-registration-company-address>
```

Comment:
Here and below:
I would like to see `(change)="onRegistrationCompanyAddressChange"`, i.e. name output as usual (`change`) and use component name in handler

Reply:
Fixed